src/views/Submit.vue

```diff
@@ -413,6 +456,29 @@ export default {
 				},
 			]
 		},
+
+		/**
+		 * Buttons for the "confirm restart submission" dialog
+		 */
+		confirmRestartSubmissionButtons() {
+			return [
+				{
+					label: t('forms', 'Abort'),
+					icon: IconCancelSvg,
+					callback: () => {},
+				},
```

Comment:
I think tertiary is enough here.
```suggestion
				{
					label: t('forms', 'Abort'),
					icon: IconCancelSvg,
					type: tertiary,
					callback: () => {},
				},
```

Reply:
ok, but we have other modals with same "Abort" button and without this `type="tertiary"`

Reply:
True... Didn't check all dialogs. Then let's keep secondary 👍🏻

Reply:
reverted to be consistent with other dialogs